Rewrite instructions for pipenv in the Readme file (#400)
RedHat-Israel/ROSE: README.md

## Requirements

First we need to make sure we have pipenv installed:

    pipenv --version

If you don't have it installed, the best way is to install it only for
your user:

    pip install --user pipenv

> ## Getting started

Reviewer: I would add that these steps should be preform only one time.

Use pipenv to create a virtual environment and to install the rest of the dependencies:

    pipenv --python /usr/local/bin/python3.7 install

You can also install packages from your distribution, but they may be
too old.

After creating the environment, we want to activate and enter our environment: 

    pipenv shell

Indication that you are inside the environment, the prompt line will look like this:

>     (ROSE) [student@vm29 ROSE]$

Reviewer: this I think I would change because it's relevant for our specific VMs
(ROSE) [student@vm29 ROSE]$
maybe we can change it to:
(ROSE) [username@hostname ROSE]$

## Running the server

> Open the virtual environment that you created in the previous step: 

Reviewer: ```suggestion
If you are not in your virtual environment, please run it:
```

>     pipenv shell

Reviewer: Is it necessary to run it again?

Author: I added it for a case if someone will want to return to the environment at a different time or session than they created the initial "getting started".

Reviewer: ok, please see my inline comment about this.
